sentry/src/main/java/io/sentry/ProfilingTransactionData.java

```diff
@@ -94,6 +112,8 @@ public boolean equals(Object o) {
         && traceId.equals(that.traceId)
         && name.equals(that.name)
         && relativeStartNs.equals(that.relativeStartNs)
+        && relativeStartCpuMs.equals(that.relativeStartCpuMs)
+        && Objects.equals(relativeEndCpuMs, that.relativeEndCpuMs)
```

Comment:
`h`: `hashCode` also needs an update. We could add some unit tests for equals and hashCode to ensure we don't make the same mistake in the future.